bindings/azure/servicebusqueues/servicebusqueues.go

```diff
@@ -345,19 +382,5 @@ func (a *AzureServiceBusQueues) Close() (err error) {
 			return err
 		}
 	}
-	a.setShutdown()
 	return nil
 }
-
-func (a *AzureServiceBusQueues) setShutdown() {
```

Comment:
These 3 methods are not necessary. We have a context and looking at whether the context has been canceled is thread-safe too.